bazel/pl_build_system.bzl

```diff
@@ -23,8 +23,19 @@ load("@rules_cc//cc:defs.bzl", "cc_binary", "cc_library", "cc_test")
 load("@rules_python//python:defs.bzl", "py_test")
 load("//bazel:toolchain_transitions.bzl", "qemu_interactive_runner")
 
+pl_boringcrypto_go_sdk = ["1.20.4"]
 pl_supported_go_sdk_versions = ["1.16", "1.17", "1.18", "1.19", "1.20"]
 
+# The last version in this list corresponds to the boringcrypto go sdk version.
+pl_all_supported_go_sdk_versions = pl_supported_go_sdk_versions + pl_boringcrypto_go_sdk
```

Comment:
In the next change I will replace `pl_supported_go_sdk_versions` with `pl_all_supported_go_sdk_versions` in order to create the container targets used in the TLS tracing tests.